pkg/services/rendering/rendering.go

```diff
@@ -406,7 +407,8 @@ func (rs *RenderingService) getNewFilePath(rt RenderType) (string, error) {
 	return filepath.Abs(filepath.Join(folder, fmt.Sprintf("%s.%s", rand, ext)))
 }
 
-func (rs *RenderingService) getURL(path string) string {
+// createURLForRendering creates a URL to send to the image rendering as callback for rendering a Grafana resource
+func (rs *RenderingService) createURLForRendering(path string) string {
```

Comment:
```suggestion
// getGrafanaURLForRendering creates a URL to send to the image rendering as callback for rendering a Grafana resource
func (rs *RenderingService) getGrafanaURLForRendering(path string) string {
```
I agree this can be improved but I was still confused with the new name so here are some suggestions to   clarify it. I think it would be nice to mention "Grafana" as this is the Grafana URL we are building and not the image renderer one (and that's the thing that can be confusing in the image renderer process).